### PR TITLE
Front-44

### PR DESCRIPTION
**Pull request by SimonjShurety**

### CODE REVIEW

File: src/components/Header/_Header.scss [hunk 1]
```diff
@@ -1,96 +1,99 @@
...
-  display: flex;
-  justify-content: space-between;
+    position: sticky;
+    z-index: 2;
+    width: 100%;
+    height: 65px;
+    padding: 0 20px;
+    display: flex;
+    justify-content: space-between;
+    background-color: #faf9f9;
```
Comment (SimonjShurety) [inline src/components/Header/_Header.scss:18]: Sorry about all the changes, my prettier config has automatically edited a bunch of stuff making it impossible to see the minor changes I made. I'll have to find a solution for that.

@koral14 I thought I had added a light grey colour to the navbar.

**Comment by SimonjShurety** [comment]
Sorry about all the changes, my prettier config has automatically edited lots of code regarding spacing and semi-colons making it almost impossible to see the minor changes I made. I'll have to find a solution for that.

**Review by koral14 — APPROVED** [review]
Great job Simon!

**Review by Ekaterina-Bondareva — APPROVED** [review]
Nica job Simon!

**Comment by Ekaterina-Bondareva** [comment]
Nice job Simon!

**Comment by SimonjShurety** [comment]
Kate, would you kindly review my merge conflict resolutions?

**Pull request merged**